src/components/templates/ProductTemplate/ProductTemplate/ProductTemplate.tsx

```diff
@@ -7,16 +7,31 @@ import { ProductTemplateHeader } from '../ProductTemplateHeader/ProductTemplateH
 import FlexContainer from '../../../layout/FlexContainer/FlexContainer';
 import FlexRow from '../../../layout/FlexRow/FlexRow';
 import FlexCol from '../../../layout/FlexCol/FlexCol';
+import styled from 'styled-components';
+import grid from '../../../../constants/grid';
 
 export interface ProductTemplateProps {
   children: React.ReactNode;
-  title: string;
+  title?: string;
   subtitle?: string;
   prevStep?: string | ReactNode;
   progress?: Pick<ProgressProps, 'currentStep' | 'totalSteps'>;
   contentWidth?: number;
 }
 
+function overlayElementAbove(marginTop: number) {
+  return `
+    margin-top: -${marginTop - 24}px; 
+    @media (min-width: ${grid.breakpoints.m}px) {
+      margin-top: -${marginTop}px; 
+    }
+    `;
+}
+
+const ContentContainer = styled.div<{ hasTitle: boolean }>(({ hasTitle }) =>
```

Comment:
provably you don't need an additional div here, you can use `style(FlexRow)` to modify the existing FlexRow